server/tests/db/test_01_es6.js

```diff
@@ -327,6 +338,132 @@ tests.NumberMaxSafeInteger = function() {
+  console.assert(Math.atanh(0.5).toPrecision(10) === '0.5493061443', 'Math.atanh 0.5');
+  console.assert(Math.atanh(1) === Infinity, 'Math.atanh 1');
+  console.assert(Number.isNaN(Math.atanh(2)), 'Math.atanh 2');
+};
+
+tests.MathCbrt = function() {
+  console.assert(Number.isNaN(Math.cbrt(NaN)), 'Math.cbrt NaN');
+  console.assert(Math.cbrt(-1) === -1, 'Math.cbrt -1');
+  console.assert(Object.is(Math.cbrt(-0), -0), 'Math.cbrt -0');
+  console.assert(Math.cbrt(-Infinity) === -Infinity, 'Math.cbrt -Infinity');
+  console.assert(Object.is(Math.cbrt(0), 0), 'Math.cbrt 0');
+  console.assert(Math.cbrt(1) === 1, 'Math.cbrt 1');
+  console.assert(Math.cbrt(Infinity) === Infinity, 'Math.cbrt Infinity');
+  console.assert(Math.cbrt(null) === 0, 'Math.cbrt null');
+  console.assert(Math.cbrt(2).toPrecision(10) === '1.259921050', 'Math.cbrt 2');
+};
+
+tests.MathClz32 = function() {
+  console.assert(Math.clz32(1) === 31, 'Math.clz32 1');
+  console.assert(Math.clz32(1000) === 22, 'Math.clz32 1000');
+  console.assert(Math.clz32() === 32, 'Math.clz32 ()');
+  console.assert(Math.clz32(NaN) === 32, 'Math.clz32 NaN');
+  console.assert(Math.clz32(Infinity) === 32, 'Math.clz32 Infinity');
+  console.assert(Math.clz32(-Infinity) === 32, 'Math.clz32 -Infinity');
+  console.assert(Math.clz32(0) === 32, 'Math.clz32 0');
+  console.assert(Math.clz32(-0) === 32, 'Math.clz32 -0');
+  console.assert(Math.clz32(null) === 32, 'Math.clz32 null');
+  console.assert(Math.clz32(undefined) === 32, 'Math.clz32 undefined');
+  console.assert(Math.clz32('foo') === 32, 'Math.clz32 "foo"');
+  console.assert(Math.clz32({}) === 32, 'Math.clz32 {}');
+  console.assert(Math.clz32([]) === 32, 'Math.clz32 []');
+  console.assert(Math.clz32(true) === 31, 'Math.clz32 true');
+  console.assert(Math.clz32(3.5) === 30, 'Math.clz32 3.5');
+};
+
+tests.MathCosh = function() {
+  console.assert(Math.cosh(-1).toPrecision(10) === '1.543080635', 'Math.cosh -1');
+  console.assert(Math.cosh(0) === 1, 'Math.cosh 0');
+  console.assert(Math.cosh(1).toPrecision(10) === '1.543080635', 'Math.cosh 1');
+};
+
+tests.MathExpm1 = function() {
+  console.assert(Math.expm1(-1).toPrecision(10) === '-0.6321205588', 'Math.expm1 -1');
+  console.assert(Math.expm1(0) === 0, 'Math.expm1 0');
+  console.assert(Math.expm1(1).toPrecision(10) === '1.718281828', 'Math.expm1 1');
+};
+
+tests.MathFround = function() {
+  console.assert(Math.fround(1.5) === 1.5, 'Math.fround 1.5');
+  console.assert(Math.fround(1.337).toPrecision(10) === '1.337000012', 'Math.fround 1.337');
+  console.assert(Math.fround(Math.pow(2, 150)) === Infinity, 'Math.fround 2**150');
+  console.assert(Number.isNaN(Math.fround('abc')), 'Math.fround "abc"');
+  console.assert(Number.isNaN(Math.fround(NaN)), 'Math.fround NaN');
+};
+
+tests.MathHypot = function() {
+  console.assert(Math.hypot(3, 4) === 5, 'Math.hypot 3, 4');
+  console.assert(Math.hypot(3, 4, 5).toPrecision(10) === '7.071067812', 'Math.hypot 3, 4, 5');
```

Comment:
Would it be useful to note that both (3, 4, 5) and (5, 12, 13) are Pythagorean triples?  (And of course that sqrt(x^2 + y^2 + z^2) = sqrt(sqrt(x^2 + y^2)^2 + z^2).)